src/xo-app/tasks/index.js

```diff
@@ -126,37 +174,30 @@ export default class Tasks extends Component {
     }
 
     const { formatMessage } = intl
+
     return (
       <Page
         header={HEADER}
         title={`(${nTasks}) ${formatMessage(messages.taskPage)}`}
       >
         <Container>
           <Row className='mb-1'>
-            <SelectPool
-              multi
-              value={state.pools}
-              onChange={this.linkState('pools')}
-            />
+            <Col mediumSize={8}>
+              <SelectPool multi onChange={this.linkState('pools')} />
+            </Col>
+            <Col mediumSize={4}>
+              <div ref={container => this.setState({ container })} />
+            </Col>
           </Row>
-          {map(
-            props.pools,
-            pool =>
-              this._showPoolTasks(pool) && (
-                <Row key={pool.id}>
-                  <Card>
-                    <CardHeader key={pool.id}>
-                      <Link to={`/pools/${pool.id}`}>{pool.name_label}</Link>
-                    </CardHeader>
-                    <CardBlock>
-                      {map(pendingTasksByPool[pool.id], task => (
-                        <TaskItem key={task.id} task={task} />
-                      ))}
-                    </CardBlock>
-                  </Card>
-                </Row>
-              )
-          )}
+          <SortedTable
```

Comment:
Wrap it in a `<Row>` > `<Col>` > `<SortedTable>` as "[Content should be placed within columns, and only columns may be immediate children of rows.](http://v4-alpha.getbootstrap.com/layout/grid/#how-it-works)".